### PR TITLE
Document config.active_job.default_queue_name [ci skip]

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Thank you for the pull request. I think we should not document global options on these guides. We have a guide only for them. @fxn WDYT?

**Comment by zzak** [comment]
I feel like this is documented already someplace?

**Comment by rafaelfranca** [comment]
Active Job doesn't have a section inside the configuration guide. Time to start it.

**Comment by eliotsykes** [comment]
@rafaelfranca Will this mean adding an Active Job section to https://github.com/rails/rails/blob/master/guides/source/configuring.md ? I'd be keen to help with this contribution.

**Comment by seuros** [comment]
@eliotsykes yes

**Comment by rafaelfranca** [comment]
Yes. It would be a great contribution.

**Comment by eliotsykes** [comment]
Thanks I'll happily take that on :+1:

File: guides/source/active_job_basics.md [hunk 1]
```diff
@@ -127,6 +127,18 @@ end
 Queues
 ------
 
+The default queue jobs are added to is named `:default`. You can change the 
+default queue using `config.active_job.default_queue_name` in `application.rb`:
```
Comment (shunsukeaida) [inline guides/source/active_job_basics.md]: Is the first sentence grammatically correct?

Comment (eliotsykes) [inline guides/source/active_job_basics.md]: Thanks @shunsukeaida - I think this PR is not going to be merged. You might be interested in reviewing this related PR: #19712

**Comment by eliotsykes** [comment]
Closing in favor of #19712

**Pull request closed without merging**